Tests.cs

```diff
@@ -0,0 +1,158 @@
+using Allure.Commons;
+using NUnit.Allure.Attributes;
+using NUnit.Allure.Core;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Training_Center_Task_15
+{
+    [AllureNUnit]
+    [AllureParentSuite("Root Suite")]
+    public class Tests
+    {
+        protected string _url = @"https://www.automationexercise.com/";
+
+        protected string _newUserBaseUsername = "TestUser";
+        protected string _username = "MyUser";
+        protected string _email = "myuser@email.com";
+        protected string _password = "123";
+        protected string _domain = "@email.com";
+
+        private IWebDriver _driver;
+        private HomePage _homePage;
+        private LoginPage _loginPage;
+        private AccountInformationPage _accountInformationPage;
+        private AccountCreatedPage _accountCreatedPage;
+        private ProductsPage _productsPage;
+        private CartPage _cartPage;
+
+        [SetUp]
```

Comment:
Tests from one test class should be executed in one browser (if 3 test classes – browser should be opened 3 times, before each test class, not before each test method)

Reply:
Added [OneTimeSetup] and [OneTimeTearDown] to each test class.